scripts/heat/devstack-heat-template.yaml

```diff
@@ -0,0 +1,164 @@
+heat_template_version: 2016-10-14
+
+description: |
+  devstack on Leap 15.0
+
+parameters:
+  key_name:
+    type: string
+    label: Key-pair name
+    description: Name of key-pair to be used for compute instance
+
+  floating_network_name:
+    type: string
+    label: Floating network name
+    description: Name of key-pair to be used for compute instance
+    default: floating
+
+  flavor_name:
+    type: string
+    label: Flavor
+    description: Flavor of the server
+    default: m1.large
+
+  image_name:
+    type: string
+    label: Image
+    description: Image to boot the server with
+    default: openSUSE-Leap-15.0
+
+  run_tempest:
+    type: boolean
+    label: Run Tempest
+    description: Set to 'yes' to run Tempest tests.  Defaults to 'no'.
+    default: False
+
+conditions:
+  skip_tempest_res:
+    equals:
+      - {get_param: run_tempest}
+      - False
+
+resources:
+  ping_ssh_secgroup:
+    type: OS::Neutron::SecurityGroup
+    properties:
+      name: ping_ssh_secgroup
+      description: Ping and SSH
+      rules:
+      - protocol: icmp
+      - protocol: tcp
+        port_range_min: 22
+        port_range_max: 22
+
+  network:
+    type: OS::Neutron::Net
+    properties:
+      port_security_enabled: True
+
+  subnet:
+    type: OS::Neutron::Subnet
+    properties:
+      network_id: { get_resource: network }
+      cidr: 10.1.1.0/24
+      ip_version: 4
+      enable_dhcp: True
+      allocation_pools:
+        - start: 10.1.1.50
+          end:   10.1.1.99
+      gateway_ip: 10.1.1.1
+
+  router_ext:
+    type: OS::Neutron::Router
+    properties:
+      external_gateway_info:
+        network: { get_param: floating_network_name }
+
+  router_ext_interface:
+    type: OS::Neutron::RouterInterface
+    properties:
+      router_id: { get_resource: router_ext }
+      subnet_id: { get_resource: subnet }
+
+  port:
+    type: OS::Neutron::Port
+    properties:
+      network: { get_resource: network }
+      fixed_ips:
+        - subnet_id: { get_resource: subnet }
+      security_groups:
+        - default
+        - { get_resource: ping_ssh_secgroup }
+
+  floating_ip:
+    type: OS::Neutron::FloatingIP
+    properties:
+      floating_network: floating
+
+  floating_ip_assoc:
+    type: OS::Neutron::FloatingIPAssociation
+    properties:
+      floatingip_id: { get_resource: floating_ip }
+      port_id: { get_resource: port }
+
+  server:
+    type: OS::Nova::Server
+    properties:
+      key_name: { get_param: key_name }
+      image: { get_param: image_name }
+      flavor: { get_param: flavor_name }
+      networks:
+        - port: { get_resource: port }
+      user_data_format: SOFTWARE_CONFIG
+      user_data:
+        get_resource: cloud_init
+
+  cloud_init:
+    type: OS::Heat::CloudConfig
+    properties:
+      # Here we use qa_devstack.sh, but see also
+      # https://wiki.microfocus.net/index.php/SUSE/ECP#DevStack_on_engcloud
```

Comment:
Remove the above two lines. Later replace the content at that link with the docs for this.

Reply:
What's your logic here?  Generally providing more info is more helpful than providing less.

Reply:
The idea was to provide one way to install devstack inside a cloud that is well used and maintained, instead of 10 documents that are all slightly different.

Reply:
Sure - when the unification is complete then it will be fine to remove references to outdated installation techniques.  But this PR is just one of several steps before we reach that point, and in the meantime I'd prefer for people to be able to easily find the alternatives.  Having said that it should be clearer which techniques are preferred vs. deprecated.